code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -251,6 +251,54 @@
 			qdel(R)
 	T.Bless()
 
+/datum/reagent/hydrogen_peroxide
+	name = "Hydrogen peroxide"
+	description = "An ubiquitous chemical substance that is composed of hydrogen and oxygen and oxygen."
```

Comment:
```suggestion
	description = "An ubiquitous chemical substance that is composed of hydrogen and oxygen."
```

Reply:
this is actually on purpose, it is a pun

Reply:
hmm!